src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -161,13 +161,55 @@
     
     @Test public void verbose() throws Exception {
         DurableTask task = new PowershellScript("$VerbosePreference = \"Continue\"; Write-Verbose \"Hello, World!\"");
+        Controller c = task.launch(new EnvVars(), ws, launcher, listener);
+        while (c.exitStatus(ws, launcher, listener) == null) {
+            Thread.sleep(100);
+        }
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        assertEquals(0, c.exitStatus(ws, launcher).intValue());
+        assertThat(baos.toString(), containsString("VERBOSE: Hello, World!"));
+        c.cleanup(ws);
+    }
+    
+    @Test public void verboseNegativeTest() throws Exception {
+        DurableTask task = new PowershellScript("$VerbosePreference = \"Continue\"; Write-Verbose \"Hello, World!\"; Write-Output \"Success\"");
         task.captureOutput();
         Controller c = task.launch(new EnvVars(), ws, launcher, listener);
         while (c.exitStatus(ws, launcher, listener) == null) {
             Thread.sleep(100);
         }
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        assertTrue(c.exitStatus(ws, launcher, listener).intValue() == 0);
+        assertThat(baos.toString(), containsString("Hello, World!"));
+        assertEquals("Success\r\n", new String(c.getOutput(ws, launcher)));
```

Comment:
does powershell on Linux use `\r\n` or just `\n` (ie will this test fail if run on Linux?)

Reply:
Good point.  I will verify this and update the PR if needed.

Reply:
PowerShell on Linux uses \n.  I'll update these tests.

Reply:
In a related note it would be nice if PowerShell core could be installed on the Linux VM's used in testing with a symlink added for powershell->pwsh, because as it stands right now the tests are skipped on Linux.

Reply:
@gabloe That can be raised as a Jenkins INFRA ticket for action by the community infra maintainers.  Be aware though that turnaround times may very because the number of people who can handle those tickets is limited and their time is heavily committed usually.